lib/ocpp/v201/charge_point.cpp

```diff
@@ -3594,18 +3594,26 @@ void ChargePoint::handle_install_certificate_req(Call<InstallCertificateRequest>
     const auto msg = call.msg;
     InstallCertificateResponse response;
 
-    const auto result = this->evse_security->install_ca_certificate(
-        msg.certificate.get(), ocpp::evse_security_conversions::from_ocpp_v201(msg.certificateType));
-    response.status = ocpp::evse_security_conversions::to_ocpp_v201(result);
-
-    if (response.status == InstallCertificateStatusEnum::Accepted) {
-        const auto& security_event = ocpp::security_events::RECONFIGURATIONOFSECURITYPARAMETERS;
-        std::string tech_info =
-            "Installed certificate: " + conversions::install_certificate_use_enum_to_string(msg.certificateType);
-        this->security_event_notification_req(CiString<50>(security_event), CiString<255>(tech_info), true,
-                                              utils::is_critical(security_event));
+    if ((msg.certificateType == InstallCertificateUseEnum::CSMSRootCertificate ||
```

Comment:
Allowing installation of `CSMSRootCertificate` and `ManufacturerRootCertificate` should be configurable, not hardcoded. I think preferably they should be configurable separately from each other.

Default should also allow this since that is what the current implementation does.

Reply:
done, I make them `ReadWrite` and `not required`